mon/entrypoint.sh

```diff
@@ -17,6 +17,17 @@ if [ ! -n "$MON_IP" ]; then
    exit 1
 fi
 
+# When using contanier network stack we use the first available ip
+if [ "$MON_IP" == "container" ]; then
+   MON_IP=$(hostname --ip-address)
```

Comment:
not sure about the "hostname --ip-address"
on my system it returns 2 addresses.

might I suggest this to get the IP?

ip -4 -o a | awk '/eth0/ { sub ("./..", "", $4); print $4 }'